mtp_api/apps/notification/rules.py

```diff
@@ -284,12 +284,13 @@ def get_profile_records_of_same_type(self, profile, record):
     ),
     'CSFREQ': CountingRule(
         'CSFREQ',
-        description='More than {limit} credits from the same debit card or bank account to any prisoner in 4 weeks',
+        description='More than {limit} credits from the same debit card or bank account to any prisoner in a week',
         abbr_description='freq. source',
         applies_to_models=(Credit,),
         profile='sender_profile',
         count='pk',
-        limit=4,
+        limit=2,
+        days=8,
```

Comment:
Are you using the dude from bodybuilding forums definition of a week?

Reply:
no idea what you mean, but the rules were always inclusive of today and the day that is 4 weeks ago. makes sense to do the same for 1 week :D

Reply:
you can google "bodybuilding forums days in a week" there are many articles about it and it's quite funny